pkg/api/const.go

```diff
@@ -187,3 +187,18 @@ const (
 	// VHDDiskSizeAKS maps to the OSDiskSizeGB for AKS VHD image
 	VHDDiskSizeAKS = 30
 )
+
+const (
+	// DefaultKubernetesCloudProviderBackoffRetries is 6, takes effect if DefaultKubernetesCloudProviderBackoff is true
+	DefaultKubernetesCloudProviderBackoffRetries = 6
+	// DefaultKubernetesCloudProviderBackoffJitter is 1, takes effect if DefaultKubernetesCloudProviderBackoff is true
+	DefaultKubernetesCloudProviderBackoffJitter = 1.0
+	// DefaultKubernetesCloudProviderBackoffDuration is 5, takes effect if DefaultKubernetesCloudProviderBackoff is true
+	DefaultKubernetesCloudProviderBackoffDuration = 5
+	// DefaultKubernetesCloudProviderBackoffExponent is 1.5, takes effect if DefaultKubernetesCloudProviderBackoff is true
+	DefaultKubernetesCloudProviderBackoffExponent = 1.5
+	// DefaultKubernetesCloudProviderRateLimitQPS is 3, takes effect if DefaultKubernetesCloudProviderRateLimit is true
```

Comment:
A few customers are complaining about set this issue too small (current is 3), are we going to increase this value?

Reply:
@jackfrancis At least in my newly installed 1.12.1 acs-engine cluster the defaults led to many errors in the kube controller like this:

```
E1028 21:32:55.584474       1 operation_generator.go:187] VolumesAreAttached failed for checking on node "k8s-agentpool2-36147952-vmss000000" with: azure - cloud provider rate limited(read) for operation:VMSSGetInstanceView
```

Upping the limits manually in `/etc/kubernetes/azure.json` and bouncing the controller pod solved the problem, but I wouldn't expect to have to do or know about this at all.

Reply:
@rocketraman how many nodes are in your cluster? what limits are working for you?

Reply:
@jackfrancis On this 1.12.1 cluster I have only 6 agent nodes. I used a bucket of 70 (instead of 10) and a qps of 6 instead of 3.